src/components/markdown-render/components/code-editor.vue

```diff
@@ -156,6 +174,13 @@
     // todo: refresh tables data and when
   }
 
+  const copy = () => {
+    navigator.clipboard.writeText(code.value)
+    Message.success({
+      content: i18n.global.t('copied'),
+      duration: 2 * 1000,
+    })
+  }
```

Comment:
Can this be like the copy icon UI (change icon after copied)? The global message is difficult for users to notice.

Reply:
Not very easy to do, arco does not provide a simple interface to do so